### PR TITLE
Gradle-ify All The Things (part 2)

### PR DESCRIPTION
**Pull request by danybony**
Migrate to gradle the build system of the remaining demo projects:
 
* MultipleContacts	 
* MultpileActivityTabHost	 
* PinchZoomDetector	 
* PushNotifications	 
* simperAudioStreamer	 
* sms	 
* TabHostExample	 
* TabHostMultipleActivityBrowsing	 
* TabHostSelfContainedTabBrowsing	 
* wizard

And update compileSdk and targetSdk versions to the last available.
screen shot 2015-01-06 at 16 01 52

### CODE REVIEW

**Comment by stefanhoth** [comment]
**Sorry, we could not display the entire diff because too many files (428) changed.*

**Comment by danybony** [comment]
Argh! Too many files moved in subdirs.. :/

**Comment by danybony** [comment]
I'll close and reopen two small PRs :disappointed:

File: CompassExample/build.gradle [hunk 1]
```diff
@@ -16,6 +16,6 @@ android {
 
 dependencies {
   compile 'com.actionbarsherlock:actionbarsherlock:4.4.0@aar'
```
Comment (stefanhoth) [inline CompassExample/build.gradle]: Wow. Really?

**Comment by danybony** [comment]
Reopening this as it is now smaller due to https://github.com/novoda/android-demos/pull/7 being merged into master

File: CompassExample/build.gradle (hunk 1 above)
Comment (danybony) [inline CompassExample/build.gradle]: Yup, that project was still using ActionbarSherlock...

**Comment by stefanhoth** [comment]
Looks good, just 2 comments:

Current build tools version is `21.1.2`.

Furthermore it would be good to streamline the `applicationId` of each demo into something like `com.novoda.demo.<sampleName>`.

**Comment by danybony** [comment]
Good point. Hold fire for merge, I'll update the build version in this one and create a new task for the applicationId, it's a good idea

**Comment by danybony** [comment]
Now this is ready to be :ship:

**Comment by stefanhoth** [comment]
LGTM

**Comment by xrigau** [comment]
nice! @danybony

**Pull request merged**